javascript/javascript2/week2/doublingOfNumber.js

```diff
@@ -0,0 +1,17 @@
+// Doubling of number
+
+/*let numbers = [1, 2, 3, 4];
+let newNumbers = [];
+
+for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 !== 0) {
+        newNumbers[i] = numbers[i] * 2;
+    }
+}
+
+console.log("The doubled numbers are", newNumbers); // [2, 6]*/
```

Comment:
Remember to clean up code. Because we use git we can safely remove commented code. Git always remembers it.

Reply:
Thanks. Done.